index.html

```diff
@@ -1,11 +1,13 @@
-<!DOCTYPE html>
+<html lang="en-US">
   <head>
```

Comment:
Please keep the `<!DOCTYPE html>` tag and the `<html lang="en-US">` tags.
Just have this
`<!DOCTYPE html>
 <html lang="en-US">
 <head> etc`